main-packages-list.txt

```diff
@@ -35,7 +35,7 @@ python3-paste-deploy >= 2.0.1-4.el8ost
 python3-pint >= 0.10.1-1.el8ost
 python3-scciclient
 python3-stevedore >= 3.2.2-0.20201009151242.274eaa6.el8
-python3-sushy
+python3-sushy == 3.4.2-0.20210125195043.aeeb416.el8
```

Comment:
please do not use '==', it's deprecated in favor of '=' and also we don't want to strictly pin a version but just provide a minimum working version